DevEdu.API/DevEdu.Business.Tests/Raiting/RaitingData.cs

```diff
@@ -0,0 +1,62 @@
+﻿
+using DevEdu.DAL.Models;
+using System.Collections.Generic;
+
+namespace DevEdu.Business.Tests
+{
+    public class RaitingData
+    {
+        public const int expectedStudentRaitingId = 13;

```

Comment:
it's a bad practice to have such values as consts
it's better to var them in each test method separately